torch/_functorch/_aot_autograd/utils.py

```diff
@@ -538,3 +538,12 @@ def call_and_expect_output_descs(fn, args):
         outs_descs,
     )
     return outs_pair
+
+
+def fn_wrappers(fn):
```

Comment:
oh, is this the helper that you referred to as `fn_stack` in the next PR?

Reply:
yes, and whoops I misremembered the name